pkg/apis/eksctl.io/v1alpha4/validation.go

```diff
@@ -1,6 +1,7 @@
 package v1alpha4
 
 import (
+	"errors"
```

Comment:
Would you mind to switch from `errors.New` to `fmt.Errorf`, so it's consistent with other packages? `errors` is usually the `github.com/pkg/errors` in all other places...

Reply:
Done. I also use `errors.New()` in tests. I'll change those as well, or those can be from the `errors` package?

Reply:
It's not a big deal, let's keep it that way.